src/addressResolvers/utils.ts

```diff
@@ -57,7 +57,7 @@ export function mapOriginalInput(
   input: string[],
   results: Record<string, string>
 ): Record<string, string> {
-  const inputLc = input.map(i => i.toLowerCase());
+  const inputLc = input.filter(a => a).map(i => i.toLowerCase());
```

Comment:
Index of `input` is being used with inputLc's index in return 

```
return withoutEmptyValues(
    Object.fromEntries(
      inputLc.map((key, index) => {
        return [input[index], resultLc[key]];
      })
    )
  );
  ```
  maybe there can be differences?

Reply:
true

Reply:
Should be better now